.eslintrc.js

```diff
@@ -1,41 +1,18 @@
 module.exports = {
   env: {
-    es6: true
+    es6: true,
   },
   extends: ['eslint:recommended', 'airbnb', 'plugin:prettier/recommended'],
   parser: 'babel-eslint',
   parserOptions: {
     ecmaFeatures: {
-      jsx: true
+      jsx: true,
     },
     ecmaVersion: 2018,
-    sourceType: 'module'
+    sourceType: 'module',
   },
   plugins: ['react-native', 'prettier'],
   rules: {
-    'no-restricted-imports': [
```

Comment:
This stuff got auto-deleted, I think when I set up eslint and Prettier on my machine--should I put it back in?

Reply:
Oh hmm - yeah it seems like it generates a new `.eslintrc.js`. Yes please do!